### ISSUE TITLE
Does not work with Gradle Wrapper  > 7.4.2

### ISSUE THREAD

**Issue by JasonTheDynamite** [skrugly/mirakle#122]
When using gradle wrapper 7.4.2 everything works fine.
After updating to 7.5 or latest: 7.5.2 stops working. App crashes with error: 

java.lang.RuntimeException: Unable to instantiate application com.package.App package com.package: java.lang.ClassNotFoundException: Didn't find class "com.package.App" on path: DexPathList[[zip file "/data/app/~~QVI1HFgZRlF8WjmT_DBUkA==/com.package-18hjwzkh7_fCB0VWn-o1nQ==/base.apk"]

**Comment by skrugly** [comment]
Probably you need to run `./gradlew clean` after upgrading Gradle and maybe do `Invalidate Caches and Restart` in Android Studio.

**Comment by JasonTheDynamite** [comment]
Tested. Does not help. Nothing changes after clean / restart & invalidate cache and combinations.
Changing back to 7.4.2-all fixes problem.

**Comment by skrugly** [comment]
Just checked and it works fine with Gradle `7.5` and `7.5.2`.
You could try to update versions of the following things as well:
1. Android Gradle Plugin `com.android.tools.build:gradle:7.2.2`
2. Builds tools version at least `31.0.0`
3. Android Studio to the latest versions. I'm using `Android Studio Dolphin | 2021.3.1 Beta 5`

**Comment by JasonTheDynamite** [comment]
I have latest version of all mentioned plugins & libraries.
"Android Studio Dolphin | 2021.3.1 Beta 5" - I do not use beta software. Especially Android studio.

**Comment by skrugly** [comment]
Will the app launch successfully if you build it without mirakle?

**Comment by JasonTheDynamite** [comment]
Yes. It will. Tested every possible option. With mirakle outputs folder is empty.

**Comment by skrugly** [comment]
You may try to ssh to remote machine and clean Gradle cache there
```
> ssh remote_machine
> sudo rm -rf ~/.gradle/caches/
```
And just in case stop gradle daemons on remote machine
```
> ssh remote_machine
> cd some_project_dir
> ./gradlew --stop 
```

**Comment by JasonTheDynamite** [comment]
tried. still does not work.

**Comment by skrugly** [comment]
Could you paste your `mirakle_init.gradle` file

**Comment by JasonTheDynamite** [comment]
[mirakle_init.txt](https://github.com/Adambl4/mirakle/files/9316667/mirakle_init.txt)
Please change extension to .gradle

**Comment by skrugly** [comment]
Just in case try the latest RC version of mirakle
`classpath 'io.github.adambl4:mirakle:1.5.3-RC-9`

**Comment by skrugly** [comment]
Is there a chance you can prepare a repo with a minimal project to reproduce the bug?
